lib/code_fund/user_impressions/user_impressions.ex

```diff
@@ -0,0 +1,34 @@
+defmodule CodeFund.UserImpressions do
+  alias CodeFund.Schema.UserImpression
+  use CodeFundWeb, :query
+
+  def count_for_last_thirty_days() do
```

Comment:
@hopsoft did you still want to attempt to bring these into a single query?

Reply:
Yes. I'll revisit this PR over the weekend.